EpicLoot/magiceffects.json

```diff
@@ -2859,6 +2859,48 @@
         "$mod_epicloot_me_slow_suffix1"
       ]
     },
+    {
+      "Type" : "DoubleMagicShot",
+      "DisplayText" : "$mod_epicloot_me_doublemagicshot",
+	  "Description" : "$mod_epicloot_me_doublemagicshot_desc",
+      "Requirements" : {
+        "AllowedItemTypes" : [
+          "Staff"
```

Comment:
I would add SkillType Elemental to this, so it doesn't roll on the protection staff or dead raisser